src/lookup-table/controllers/ImportCsvUpdate.controller.ts

```diff
@@ -0,0 +1,47 @@
+import { parseStream } from 'fast-csv'
+import { LookupTableService, LookupTableStructureService } from '../services'
+
+const ImportCsvUpdateController = async (request: any, reply: any) => {
+  const lookupTableId = Number(request.params.lookupTableId)
+  const data = await request.file()
+  let lookupTableDbStructure: any = {}
+
+  const lookupTableStructure = LookupTableStructureService()
+  const lookupTableService = LookupTableService()
+
+  try {
+    lookupTableDbStructure = await lookupTableStructure.getById(lookupTableId)
+
+    let stream = await parseStream(data.file, {
+      headers: lookupTableStructure.parseCsvHeaders,
+    })
+      .on('headers', (_) => {
+        lookupTableStructure.compareFieldMaps(
+          lookupTableDbStructure.fieldMap,
+          lookupTableStructure.csvFieldMap
+        )
+      })
+      .on('data', async (row: any) => {
+        stream.pause()
+        // TODO: Row ID validation, if row has id and that does not exist in database
+        await lookupTableStructure.addToCsvRows(row, lookupTableDbStructure)
```

Comment:
> Not sure about your preference for naming variables with underscore, I can't quite see consistency and not sure why it's needed, it certaintly makes the code less readable. (i.e. in `LookupTableService`, `_`lookupTableModal, in `LookupTableStructureService`, lookupTableStructureModel, with no underscore).

@andreievg the underscore is to denote a private variable in the function. Yes, they can be used without underscore and still will be private if we do not return them, but I thought it would be a bit easier for the developers to understand that they are private.